testing/tests/DevExpress.ui.events/contextmenu.tests.js

```diff
@@ -96,3 +113,35 @@ QUnit.test('contextmenu should be unsubscribed on unsubscribing contextmenu', fu
 
     assert.equal(holdEvents.length, 0, 'contextmenu event handler was removed');
 });
+
+[
+    { isSimulator: true, description: 'default\'s contextmenu should be prevented on dxhold in simulator' },
+    { isSimulator: false, description: 'default\'s contextmenu should not be prevented on dxhold' }
```

Comment:
>default\'s contextmenu should not be prevented on dxhold

why? 
"DevExtreme menu appears while I hold my finger on the screen. As soon as I remove my finger - the browser menu appears" - it looks like a "dxhold" occurs (HOLD_TIMEOUT = 750;)